### PR TITLE
Add a scap remediation feature test

### PR DESCRIPTION
**Pull request by lhellebr**
### Problem Statement
https://issues.redhat.com/browse/SAT-23240

### Solution
Add an entity in airgun and test through WebUI (this is a WebUI-only feature). Create a host, run a scan, remediate an issue.

Requires https://github.com/SatelliteQE/airgun/pull/1441

### CODE REVIEW

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation
airgun: 1441

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation
airgun: 1441

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7503
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation --external-logging
Test Result : ================= 1 passed, 16 warnings in 1305.16s (0:21:45) ==================
```

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation
airgun: 1441

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7526
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation --external-logging
Test Result : ================= 1 passed, 16 warnings in 1324.84s (0:22:04) ==================
```

**Review by pondrejk — APPROVED** [review]

File: tests/foreman/longrun/test_oscap.py [hunk 1]
```diff
@@ -230,6 +233,127 @@ def test_positive_oscap_run_via_ansible(
...
+):
+    """Run an OSCAP scan and remediate through WebUI
+
+    :id: 55b919ef-432f-4186-b22a-01bb8ce39b3f
+
+    :setup: scap content, scap policy, host group associated with the policy
+
+    :steps:
+
+        1. Create a valid scap content
```
Comment (vijaysawant) [inline tests/foreman/longrun/test_oscap.py:285]: I think, this step need to remove as this is part of setup.

File: tests/foreman/longrun/test_oscap.py [hunk 2]
```diff
@@ -230,6 +233,127 @@ def test_positive_oscap_run_via_ansible(
...
+
+@pytest.mark.e2e
+@pytest.mark.tier4
+def test_positive_oscap_remediation(
+    module_org, default_proxy, content_view, lifecycle_env, target_sat, scap_prerequisites
+):
+    """Run an OSCAP scan and remediate through WebUI
+
+    :id: 55b919ef-432f-4186-b22a-01bb8ce39b3f
+
```
Comment (vijaysawant) [inline tests/foreman/longrun/test_oscap.py:280]: This token can be added
```suggestion
:Verifies: SAT-23240
```

File: tests/foreman/longrun/test_oscap.py [hunk 3]
```diff
@@ -230,6 +233,127 @@ def test_positive_oscap_run_via_ansible(
...
+        assert result == '1'
+        # Run the actual oscap scan on the vm/clients and
+        # upload report to Internal Capsule.
+        vm.execute_foreman_scap_client()
+        arf_id = target_sat.cli.Arfreport.list({'search': f'host={vm.hostname.lower()}'})[0]['id']
+
+        # Remediate
+        with target_sat.ui_session() as session:
+            assert (
+                vm.execute("rpm -q aide").status != 0
```
Comment (vijaysawant) [inline tests/foreman/longrun/test_oscap.py:332]:
```suggestion
                vm.execute('rpm -q aide').status != 0
```

**Review by pnovotny — APPROVED** [review]

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation
airgun: 1441

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7593
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/longrun/test_oscap.py::test_positive_oscap_remediation --external-logging
Test Result : ================= 1 passed, 16 warnings in 1286.53s (0:21:26) ==================
```

**Pull request merged**